src/server/json_family.cc

```diff
@@ -157,6 +159,48 @@ OpResult<json> GetJson(const OpArgs& op_args, string_view key) {
   return decoder.get_result();
 }
 
+string ConvertToJsonPointer(const string& json_path) {
+  if (json_path.empty() || json_path[0] != '$') {
+    return "";
+  }
+
+  vector<string> parts;
```

Comment:
you can use string_view to reference the original string - saves allocations

Reply:
in fact if you use string_view instead of `const string& json_path` in function declaration, you won't need to change anything and it will work naturally with string_views

Reply:
I probably was not clear. You still need to use `vector<string_view>`, otherwise you will still allocate strings.